ast/ddl.go

```diff
@@ -821,7 +821,34 @@ type CreateViewStmt struct {
 
 // Restore implements Node interface.
 func (n *CreateViewStmt) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	ctx.WriteKeyWord("CREATE ")
+	if n.OrReplace {
+		ctx.WriteKeyWord("OR REPLACE ")
+	}
+	ctx.WritePlainf("ALGORITHM = %s ", n.Algorithm.String())
+	if n.Definer.CurrentUser {
+		ctx.WritePlain("DEFINER = current_user ")
+	} else {
+		ctx.WritePlain("DEFINER = ")
+		ctx.WriteName(n.Definer.Username)
+		if n.Definer.Hostname != "" {
+			ctx.WritePlain("@")
+			ctx.WriteName(n.Definer.Hostname)
+		}
+	}
+	ctx.WritePlainf(" SQL SECURITY %s ", n.Security.String())
+	ctx.WriteKeyWord("VIEW ")
+	ctx.WriteName(n.ViewName.Schema.O)
+	ctx.WritePlain(".")
+	ctx.WriteName(n.ViewName.Name.O)
+	ctx.WriteKeyWord(" AS ")
+
+	if err := n.Select.Restore(ctx); err != nil {
+		return errors.Annotate(err, "An error occurred while create index")
+	}
+
+	ctx.WritePlainf(" WITH %s CHECK OPTION", n.CheckOption.String())
+	return nil
 }
```

Comment:
switch the test case on in `parser_test.go -> TestView`, eg:
```
{"create view v as select * from t", true, "CREATE VIEW `v` AS SELECT * FROM `t`"},
```

Reply:
and I think we should wait for the implement of `SelectStmt` .

Reply:
what about trying to implement `Restore` of `AlterTableStmt`?

Reply:
@leoppro I think you can merge this PR after `SelectStmt` is complete.

Reply:
And wildcard in `SelectStmt` should be explain with column and column names,

Reply:
> And wildcard in `SelectStmt` should be explain with column and column names, If we can explain this in AST level?

no, we use `WildCardField` to mark it.

Reply:
That would be a problem for me...